lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -144,6 +157,15 @@ def parse_house_number(address)
         house.empty? ? nil : house
       end
 
+      def add_wallet(post, wallet, action, options = {})
```

Comment:
this is for subscriptions right? do all wallets support subscription? if not how do we differentiate?
cc: @inai-vijay